apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -215,9 +216,9 @@ type ImportYamlOutput struct {
 
 type Capabilities struct {
 	LoadBalancerCapabilities LoadBalancerCapabilities `json:"loadBalancerCapabilities,omitempty"`
-	IngressCapabilities      []IngressCapabilities    `json:"ingressCapabilities,omitempty"`
-	NodePoolScalingSupported bool                     `json:"nodePoolScalingSupported,omitempty"`
-	NodePortRange            string                   `json:"nodePortRange,omitempty"`
+	IngressCapabilities      []IngressCapabilities    `json:"ingressCapabilities,omitempty" norman:"noupdate"`
+	NodePoolScalingSupported bool                     `json:"nodePoolScalingSupported,omitempty" norman:"noupdate"`
+	NodePortRange            string                   `json:"nodePortRange,omitempty" norman:"noupdate"`
 }
```

Comment:
noupdate will disallow users from updating the other capabilities. But we probably need to disallow users from setting them during cluster creation as well?

Reply:
Added the nocreate tag